Introduce `random` JsonPath function (#4)
vividus-framework/JsonPath: README.md

| `last()`    | Provides the last item of an array                                                   | Depends on the array |
| `index(X)`  | Provides the item of an array of index: X, if the X is negative, take from backwards | Depends on the array |
| `distinct()`| Provides an array containing only unique items from the input array                  | List<E>              |

> | `random()`  | Provides the random item of an array                                                 | Depends on the array |

Reviewer: ```suggestion
| `random()`  | Provides one random item from the array                                                 | Depends on the array |
```

> >>>>>>> Stashed changes

Reviewer: ```suggestion
```
